introduction.md

```diff
@@ -1,2 +1,7 @@
 This is an introduction
-ddJavascript Path
+Javascript Path
+
+Deadly Skills
+* Git
+* JS
+* Beer
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?